jupyter_spark/spark.py

```diff
@@ -21,18 +21,18 @@ def initialize(self, host, url, prefix):
         self.full_url = url_path_join(self.url, self.prefix)
 
     def get(self):
-        if not self.request.uri.startswith(self.prefix):
```

Comment:
I was wondering about that already but couldn't for the live of me not find what the Tornado request uri contains, just a URI path or the full URI. This clears it up :)